src/Docker.jl

```diff
@@ -125,7 +126,16 @@ function build_executor_command(exe::DockerExecutor, config::SandboxConfig, user
     end
 
     # Begin building `docker` args
-    cmd_string = String["docker", "run", "--privileged", "-i", "--label", docker_image_label(exe)]
+    if (exe.privileges === :privileged) || (exe.privileges === nothing) # this is the default
+        privilege_args = String["--privileged"]
+    elseif exe.privileges === :no_new_privileges
+        privilege_args = String["--security-opt", "no-new-privileges"]
```

Comment:
Would be good to include some comments about when a user might want to use `--privileged` versus `--security-opt no-new-privileges`.

Reply:
Done!